src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -747,11 +803,9 @@ def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy=""
 
     # Get kubernetes cluster info for telemetry
     kubernetes_version = get_server_version(configuration)
-    kubernetes_distro = get_kubernetes_distro(configuration)
 
     kubernetes_properties = {
-        'Context.Default.AzureCLI.KubernetesVersion': kubernetes_version,
-        'Context.Default.AzureCLI.KubernetesDistro': kubernetes_distro
```

Comment:
Why is this removed?

Reply:
Because, in update_agents we havent exposed parameters, and so, heuristics will set the distro and infra. But, if user has already provided the distro and infra through --distribution and --infrastructure, this will skew the helm parameters and telemetry with respect to the cc object. And we dont patch these distro/infra values in CC object now, as it was decided. kubernetes_version was removed by mistake I think, will replace it.

Reply:
Cant we do a get on the cc resource and use the distro value in telemetry?

Reply:
Yeah, that sounds good.